src/mono/wasm/runtime/loader/polyfills.ts

```diff
@@ -19,16 +23,30 @@ const URLPolyfill = class URL {
-    mono_assert(typeof globalThis.BigInt64Array === "function", "This browser/engine doesn't support BigInt64Array API. Please use a modern version.");
+    mono_assert(ENVIRONMENT_IS_SHELL || typeof globalThis.URL === "function", "This browser/engine doesn't support URL API. Please use a modern version. See also https://aka.ms/dotnet-wasm-features");
+    mono_assert(typeof globalThis.BigInt64Array === "function", "This browser/engine doesn't support BigInt64Array API. Please use a modern version. See also https://aka.ms/dotnet-wasm-features");
     if (MonoWasmThreads) {
-        mono_assert(!ENVIRONMENT_IS_SHELL && !ENVIRONMENT_IS_NODE, "This build of dotnet is multi-threaded, it doesn't support shell environments like V8 or NodeJS.");
-        mono_assert(globalThis.SharedArrayBuffer !== undefined, "SharedArrayBuffer is not enabled on this page. Please use a modern browser and set Cross-Origin-Opener-Policy and Cross-Origin-Embedder-Policy http headers. See also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer#security_requirements");
-        mono_assert(typeof globalThis.EventTarget === "function", "This browser/engine doesn't support EventTarget API. Please use a modern version.");
+        mono_assert(!ENVIRONMENT_IS_SHELL && !ENVIRONMENT_IS_NODE, "This build of dotnet is multi-threaded, it doesn't support shell environments like V8 or NodeJS. See also https://aka.ms/dotnet-wasm-features");
+        mono_assert(globalThis.SharedArrayBuffer !== undefined, "SharedArrayBuffer is not enabled on this page. Please use a modern browser and set Cross-Origin-Opener-Policy and Cross-Origin-Embedder-Policy http headers. See also https://aka.ms/dotnet-wasm-features");
+        mono_assert(typeof globalThis.EventTarget === "function", "This browser/engine doesn't support EventTarget API. Please use a modern version. See also https://aka.ms/dotnet-wasm-features");
     }
+}
 
-    // TODO detect other (WASM) features that are required for the runtime
-    // See https://github.com/dotnet/runtime/issues/84574
+export async function verifyEnvironmentAsync() {
+    if (ENVIRONMENT_IS_NODE) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore:
+        const process = await import(/* webpackIgnore: true */"process");
+        if (process.versions.node.split(".")[0] < 14) {
+            throw new Error(`NodeJS at '${process.execPath}' has too low version '${process.versions.node}'`);
+        }
+    }
+    if (WasmEnableSIMD) {
+        mono_assert(await simd(), "This browser/engine doesn't support WASM SIMD. Please use a modern version.");
+    }
+    if (WasmEnableExceptionHandling) {
+        mono_assert(await exceptions(), "This browser/engine doesn't support WASM exception handling. Please use a modern version.");
```

Comment:
Should these two also have the link to `dotnet-wasm-features`?

Reply:
heh, this was reason why I created the link but forgot to update this place, thanks!